website/content/en/status/report-2023-01-2023-03/portmgr.adoc

```diff
@@ -16,7 +16,7 @@ Below is what happened in this quarter.
 Currently we have around 33,500 ports in the tree.
 For these ports, there are 3,021 open problem reports, of which 764 are unassigned.
 The first three months of this year saw 9,021 commits by 163 committers for the `main` branch and 701 commits by 55 committers for the `2023Q1` branch.
-Compared to 2022Q4, this means a slight increase in the number of ports, port PRs, ports commits, and active port committers.
+Compared to `2022Q4`, this means a slight increase in the number of ports, port PRs, ports commits, and active port committers.
```

Comment:
… if this _looks_ weird, revert it. 

(2023Q1 describes a branch. 2022Q4 describes a quarter.)